dbld/images/debian-bullseye.dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM debian:bullseye
+LABEL maintainer="Andras Mitzki <andras.mitzki@balabit.com>, Laszlo Szemere <laszlo.szemere@balabit.com>, Balazs Scheidler <balazs.scheidler@oneidentity.com>"
+ENV OS_DISTRIBUTION=debian
+ENV OS_DISTRIBUTION_CODE_NAME=bullseye
+
+ARG ARG_IMAGE_PLATFORM
+ARG COMMIT
+ENV IMAGE_PLATFORM ${ARG_IMAGE_PLATFORM}
+LABEL COMMIT=${COMMIT}
+
+ENV DEBIAN_FRONTEND=noninteractive
+ENV DEBCONF_NONINTERACTIVE_SEEN=true
+ENV LANG C.UTF-8
+
+COPY images/fake-sudo.sh /usr/bin/sudo
+COPY images/entrypoint.sh /
+COPY . /dbld/
+
+RUN /dbld/builddeps install_dbld_dependencies
+RUN /dbld/builddeps install_apt_packages
+RUN /dbld/builddeps install_debian_build_deps
+RUN /dbld/builddeps install_criterion
```

Comment:
Just an idea for the future: we can start using `criterion` from system packages.
It's included in Debian Bullseye, Debian Testing, and in Ubuntu 21.04 and 21.10.
https://repology.org/project/criterion/versions

It's not used in syslog-ng's debian/control file yet
https://github.com/gcsideal/syslog-ng-debian/blob/debian/unstable/debian/control

Reply:
Good idea! I will look into it. Please do not merge this PR until that.

Reply:
I added it to `packaging/debian/control` :)

Reply:
superb :)